lib/GridItem.jsx

```diff
@@ -1,18 +1,16 @@
 // @flow
-import {default as React, Component} from 'react';
-import {perc, setTransform} from './utils';
+import React from 'react';
 import Draggable from 'react-draggable';
 import {Resizable} from 'react-resizable';
-import assign from 'object-assign';
+import {perc, setTransform} from './utils';
 
-import type {Position, Size} from './utils';
+import type {Position} from './utils';
 
 /**
  * An individual item within a ReactGridLayout.
  */
-export default class GridItem extends Component {
-
-  static propTypes = {
+const GridItem = React.createClass({
```

Comment:
After digging deeper into the code I saw no win in using ES6 styled classes for all components. Imho the code looks much more clean with this solution. But I would definitely not insist on this. Moreover I don't know the impact on future flow type checks where this could be contra productive.